aikau/src/main/resources/alfresco/upload/css/UploadMonitor.css

```diff
@@ -56,6 +55,17 @@
       &__action {
          display: none;
       }
+      &__progress-cell {
+         font-size: 0;
+         padding: 0;
+      }
+      &__progress-bar {
+         background: #6bf;
```

Comment:
Background, shadow and height should definitely be LESS variables. I can see this being something that SFS will want to tweak.